### PR TITLE
Add support for `pause_collection` on `Subscription`

### PR DESCRIPTION
**Pull request by remi-stripe**
Codegen for openapi cadc76e

r? @ob-stripe 
cc @stripe/api-libraries

### CODE REVIEW

File: types/2020-03-02/Issuing/Disputes.d.ts [hunk 1]
```diff
@@ -18,24 +18,24 @@ declare module 'stripe' {
         /**
          * Disputed amount. Usually the amount of the `disputed_transaction`, but can differ (usually because of currency fluctuation or because only part of the order is disputed).
          */
-        amount: number;
+        amount?: number;
```
Comment (ob-stripe) [inline types/2020-03-02/Issuing/Disputes.d.ts]: Why did all dispute fields except `id` and `object` become optional?

File: types/2020-03-02/PaymentMethods.d.ts [hunk 2]
```diff
@@ -126,7 +126,7 @@ declare module 'stripe' {
         exp_year: number;
 
         /**
-         * Uniquely identifies this particular card number. You can use this attribute to check whether two customers who've signed up with you are using the same card number, for example.
+         * Uniquely identifies this particular card number. You can use this attribute to check whether two customers who've signed up with you are using the same card number,for example. For payment methods that tokenize card information (Apple Pay, Google Pay), the tokenized number might be provided instead of the underlying card number.
```
Comment (ob-stripe) [inline types/2020-03-02/PaymentMethods.d.ts]: That comment change doesn't seem correct.

File: types/2020-03-02/Issuing/Disputes.d.ts (hunk 1 above)
Comment (remi-stripe) [reply types/2020-03-02/Issuing/Disputes.d.ts]: Internal reasons mostly

File: types/2020-03-02/PaymentMethods.d.ts (hunk 2 above)
Comment (remi-stripe) [reply types/2020-03-02/PaymentMethods.d.ts]: Why isn't this correct? It's not greatly worded, I agree, but it just means we calculate the fingerprint based on the DPAN

Comment (ob-stripe) [reply types/2020-03-02/PaymentMethods.d.ts]: We should fix the comment to be clearer, but that's not a blocker for this PR.

**Pull request merged**